apps/anoma_lib/lib/noun/nounable.ex

```
@@ -81,9 +60,21 @@ defimpl Nounable, for: BitString do
 end
 
 defimpl Nounable, for: Atom do
+  import Noun
+
   def to_noun(true), do: 0
   def to_noun(false), do: 1
   def to_noun(atom), do: Atom.to_string(atom)
```

Delete this line, let it fail on a general atom. Not the ideal option but honestly probably the best we can do, i.e. assuming we never actually compute non-boolean atoms (maybe add in the `nil` compilation to 0 or something as Ray suggested).

```suggestion
```